src/main/java/com/puppycrawl/tools/checkstyle/checks/ArrayTypeStyleCheck.java

```diff
@@ -27,9 +27,16 @@
 /**
  * Checks the style of array type definitions.
  * Some like Java-style: {@code public static void main(String[] args)}
- * and some like C-style: public static void main(String args[])
+ * and some like C-style: {@code public static void main(String args[])}.
  *
- * <p>By default the Check enforces Java style.
+ * <p>By default the Check enforces Java style.</p>
+ *
+ * <p>This check also enforces only Java style for method return types.</p>
+ * <pre>
+ * public byte getData()[] // violation
```

Comment:
This goes back to my previous comment at https://github.com/checkstyle/checkstyle/pull/6428#discussion_r257478748 ,
> Java code with violation and ok messages should be in `Examples` section.

I am fine with a small example in this section, not stating violation/ok, like was done above with `main` method.